pkg/install/install.go

```diff
@@ -126,6 +126,10 @@ func Serving() error {
 		return fmt.Errorf("wait: %w", err)
 	}
 
+	if err := reduceResources(); err != nil {
+		return fmt.Errorf("reduce: %w", err)
+	}
+
```

Comment:
should we also run this for eventing?  otherwise, I think this is looking pretty good